Header header
float64 northAcc
float64 eastAcc
float64 diuAcc
float64 nVelStaDev
float64 eVelStaDev
float64 dVelStaDev